ProjectLighthouse/PlayerData/APIKey.cs

```diff
@@ -0,0 +1,21 @@
+﻿
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LBPUnion.ProjectLighthouse.PlayerData
+{
+    public class APIKey
```

Comment:
Its worth noting that it would probably be helpful to link this key to a particular user for audit logging.

Reply:
So like a created by?

Reply:
Yeah, exactly.

Reply:
while I agree, I think that's out of the scope of this PR, I had a bash at implementing that and it's a lot of changes to the command system.